src/bench.h

```diff
@@ -121,4 +121,13 @@ int have_flag(int argc, char** argv, char *flag) {
     return 0;
 }
 
+int get_iters(int default_iters) {
+    char* env = getenv("ITERS");
```

Comment:
maybe change it to something with more meaning and less likely to collide with other software, e.g., `SECP256K1_BENCH_ITERS`

Reply:
even though it's just the for the benchmarks? we also have `int have_flag(..);` and `void print_number(..);` in that header.

Reply:
I'm talking about the environment variable, not the function. :)

Reply:
Done